_pages/01-applicants.md

```diff
@@ -25,51 +25,90 @@ For additional questions, please email <a href="mailto:oceanhkw@uw.edu" target="
+**Q: I am an undergrad / first year grad / faculty / etc. Is this program right for me?**
 
-**2020-4-23.** We are monitoring the COVID-19 situation and are in ongoing discussions 
-about its impact on plans for a virtual event in 2020 and in-person events in 2021. 
-Please check this web site regularly for updates.
-To stay up to date on the current impact of the coronavirus emergency and government recommendations,
-see the US Federal web site <a href="https://www.coronavirus.gov">https://www.coronavirus.gov</a> 
-and your state and local positions.
+We welcome participants from all career stages and encourage applications from graduate students,
+postdocs and early career researchers.
 
-**Q: I am an undergrad / first year grad / faculty / etc. Is this program right for me?**
+Before applying consider what you can bring to the even:
+by pitching a project,
+by your knowledge of data sets,
+by your programming/computational skills,
+by your project management skills.
+
+We want you to grow/learn during the event!
+If in doubt,
+simply apply and explain your motivation for participation.
 
-We welcome participants from all career stages. We strongly encourage applications from graduate students, postdocs and early career researchers. There are different ways to contribute to the event: by pitching a project, by your knowledge of data sets, by your computational skills, by your project management skills. We want you to grow/learn during the event! We expect all participants to be engaged in the team projects and focus during the week. If in doubt, simply apply and explain your motivation for participation.
+We expect all participants to be engaged in the team projects and focus during the week.
 
 **Q: I am proficient in Matlab / R / other languages but have not used Python. It has been on my list to learn. How can I make my application be stronger?**
 
-We want to build an inclusive Oceanography community regardless of people’s language of choice and programming level. Since the event is only one week long, the tutorials will be presented in one language (Python), and participants will benefit the most if they have basic familiarity with this language. We expect that you have gone through the [Software Carpentry Python tutorial](http://swcarpentry.github.io/python-novice-gapminder) to get familiar with the syntax in advance of the program. The more experience you can obtain before the beginning of the program, the easier it will be to follow the tutorials and contribute to the projects.
+We want to build an inclusive Oceanography community regardless of people’s language of choice and programming level.
+Note that most of the tutorials will be presented in Python,
+and participants will benefit the most if they have basic Python skills.
+
+We expect that you have gone through the [Software Carpentry Python tutorial](http://swcarpentry.github.io/python-novice-gapminder) to get familiar with the syntax in advance of the program.
```

Comment:
For consistency I think we should copy and paste here (replace these two sentences) the relevant text from the application form, in the description of the Programming Skills section.